src/node_messaging.cc

```diff
@@ -129,7 +131,7 @@ class DeserializerDelegate : public ValueDeserializer::Delegate {
  private:
   Environment* env_;
   const std::vector<BaseObjectPtr<BaseObject>>& host_objects_;
-  const std::vector<Local<SharedArrayBuffer>>& shared_array_buffers_;
+  const std::span<Local<SharedArrayBuffer>>& shared_array_buffers_;
```

Comment:
Some lines are changed to LocalVector some of them to std:span. It's hard to keep track the reason

Reply:
LocalVector is the storage. The `std::span` is just a view. In some cases the spans are views over a LocalVector, in other cases the spans are views over an array.